marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobManager.java

```diff
@@ -85,6 +101,153 @@ public JobDeleteResponse deleteJobs(String jobIds) {
         return this.jobDeleteRunner.deleteJobs(jobIds);
     }
 
+    /**
+     * Export Job documents and their associated Trace documents to a zip file.
+     *
+     * @param exportFilePath specifies where the zip file will be written
+     * @param jobIds a comma-separated list of jobIds; if null, all will be exported
+     */
+    public void exportJobs(Path exportFilePath, String jobIds) {
+        File zipFile = exportFilePath.toFile();
+        WriteToZipConsumer zipConsumer = new WriteToZipConsumer(zipFile);
+
+        String[] jobsArray = jobIds != null ? jobsArray = jobIds.split(",") : null;
```

Comment:
does this work?

Shoudn't it be:

`String[] jobsArray = jobIds != null ? jobIds.split(",") : null;`

Reply:
oddly, it appears to have worked, but I'll fix it